ext/tiny_tds/extconf.rb

```diff
@@ -21,9 +21,11 @@ def do_help
 do_help if arg_config('--help')
 
 # Make sure to check the ports path for the configured host
-host = RbConfig::CONFIG['host']
+architecture = RbConfig::CONFIG['arch']
+architecture = "x86-mingw32" if architecture == "i386-mingw32"
+
 project_dir = File.expand_path("../../..", __FILE__)
-freetds_ports_dir = File.join(project_dir, 'ports', host, 'freetds', FREETDS_VERSION)
+freetds_ports_dir = File.join(project_dir, 'ports', architecture, 'freetds', FREETDS_VERSION)
```

Comment:
I don't completely understand the swap from "host" to "architecture", other than that I generally agree the latter makes sense. Is switching this matching rake compiler patterns, or is it something bespoke for this project's config?

Reply:
not really on us, but rather on rake-compiler-dock.

I edited the description, so it's hopefully more clear.